engine/src/main/java/io/zeebe/engine/processing/job/JobEventProcessors.java

```diff
@@ -51,7 +48,5 @@ public void onRecovered(final ReadonlyProcessingContext context) {
                 jobState.setJobsAvailableCallback(onJobsAvailableCallback);
               }
             });
-
-    return jobErrorThrownProcessor;
```

Comment:
Yes, you are right. This is also more of a temporary measure. I talked to Phil about resolving incidents. And this will be refactored in a later ticket. So far now we kept the mechanism close to the original. That is, the processor is used directly while resolving the incident. The change here was to simplify the code already. Before, the job processor was created here and then passed around. Now, it is no longer needed here. In fact, the class shall be removed at some point. So we can construct it directly where it is used.